lib/ui/semantics.dart

```diff
@@ -488,33 +488,39 @@ class SemanticsUpdateBuilder extends NativeFieldWrapperClass2 {
     String increasedValue,
     String decreasedValue,
     TextDirection textDirection,
-    int hitTestPosition,
     Float64List transform,
-    Int32List children,
+    // TODO(yjbanov): remove after moving the framework to the new param names.
+    @deprecated Int32List children,
+    Int32List childrenInTraversalOrder,
```

Comment:
Can you document the new parameters in the method doc above?

Reply:
Done. Also removed the `@deprecated` annotation. Otherwise if someone other than myself attempts to sync the engine, they will be forced to migrate to the new system, which I already have a PR for.